entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/EntityRecordService.java

```diff
@@ -1102,7 +1103,11 @@ private void updateEntityAggregatesList(
   }
 
   private void setEuropeanaMetadata(
-      Entity europeanaProxyMetadata, String entityId, EntityRecord entityRecord, Date timestamp) {
+      Entity europeanaProxyMetadata,
+      String entityId,
+      EntityRecord entityRecord,
+      Entity datasourceResponse,
```

Comment:
dataSourceResponse should not be used as parameter, eventually the id needs to be added as parameter